app/controllers/evidence_controller.rb

```diff
@@ -6,11 +6,13 @@ class EvidenceController < NestedNodeResourceController
   include MultipleDestroy
   include NodesSidebar
   include NotificationsReader
+  include Previewable
 
-  before_action :set_or_initialize_evidence, except: [ :index, :create_multiple ]
+  before_action :set_or_initialize_evidence, only: [ :show, :new, :create, :edit, :update, :destroy ]
```

Comment:
Is this what's causing the later smell (L114)? And if so, why is this the right pattern?

Reply:
This change was made in [peer review](https://github.com/dradis/dradis-ce/pull/1204#discussion_r1403067494) to use a safer whitelist approach